docs/swagger/spec.yml

```diff
@@ -321,6 +476,89 @@ definitions:
         type: string
         description: URL safe version of name
         readOnly: true
+  LayerSource:
+    allOf:
+      - $ref: '#/definitions/BaseModel'
```

Comment:
I don't think this includes a "createdBy" or "modifiedBy" -- I think we'll want that here.